uint8 command   # message types defined in network.hpp
int8 param1 -1  # general-purpose parameter
int8 param2 -1  # general-purpose parameter

# Unused:
# string drone_id 
# setpoint # expand to leader needs
---
uint8 ack       # acknowledgment; success = 1, fail = 0
